### ISSUE TITLE
Missing "leisure=pitch" tag for Table Tennis POI

### ISSUE THREAD

**Issue by fho** [osmandapp/Osmand#3464]
When I add a new POI and choose the label "Table tennis", Osmand tags the POI only with "sport=table_tennis.

wiki.openstreetmap.org (https://wiki.openstreetmap.org/wiki/Tag:sport%3Dtable_tennis) says:
> For a public table tennis table set a node Node and add leisure=pitch + sport=table_tennis.

Therefore Osmand should set these both tags instead of only sport=table_tennis.

I'm using OsmAnd+ 2.5.4.

**Comment by vshcherb** [comment]
I'm afraid OsmAnd is not fully functional editor in that case  and on the other hand this missing tags could be always added by the OSM bot if they are necessary.

**Comment by letypequividelespoubelles** [comment]
I just had this issue too. I guess the issue is linked with #5958 

The problem is : one adds guidepost or table tennis POI to OSM via OSMAnd, then with live update dl the map, but still can't see it on the map because it misses part of the tags. Were you able to understand why it doesn't work ? 

Thanks a lot

**Comment by Chumva** [comment]
It should be added in poi_types.xml for example here - 
https://github.com/osmandapp/OsmAnd-resources/commit/2ce525936449461269eca6e64be836358d6e7a84.

**Comment by xmd5a2** [comment]
As Victor mentioned above OsmAnd is not a fully functional editor and it doesn't support tag presets.

**Comment by Chumva** [comment]
I agree, but one additional tag is possible to add via edit_tag2 edit_value2.

**Comment by xmd5a2** [comment]
Problem is in fact that `sport` tags in OSM are actually *additional* and can be used not only with `leisure=pitch` but with `leisure=stadium`, `leisure=sports_centre` and so on. However now in OsmAnd `sport` tags are *main* tags. This was done because in early OSM years very very often `sport` was used without any *main* tags mentioned above. We couldn't automatically add `leisure=pitch` to `sport` because sometimes this will be false data.
But as far as I see situation was changed and more than 93% of `sport` objects **have** *main* tags. So we need to think how to migrate `sport` tags from *additional* to *main* keeping backward compatibility.
But adding a `guidepost` via preset **will** automatically add `tourism=information` tag. I just checked that.
![Screenshot_2021-04-06-20-35-47-523_net osmand plus](https://user-images.githubusercontent.com/5829014/113754395-db0c7200-9717-11eb-96c6-19179b41fe3e.jpg)
